src/main/java/ch/njol/skript/expressions/ExprTernary.java

```diff
@@ -41,16 +41,16 @@
 
 @Name("Ternary")
 @Description("A shorthand expression for returning something based on a condition.")
-@Examples({"set {points} to 500 if {admin::%player's uuid%} is set else 100"})
+@Examples({"set {points} to 500 if {admin::%player's uuid%} is set else 100", "set {points} to (player's name = \"Notch\") ? 500 : 100"})
 @Since("2.2-dev36")
 @SuppressWarnings("null")
 public class ExprTernary<T> extends SimpleExpression<T> {
-
+	
 	static {
 		Skript.registerExpression(ExprTernary.class, Object.class, ExpressionType.COMBINED,
-				"%objects% if <.+>[,] (otherwise|else) %objects%");
+			"%objects% if <.+>[,] (otherwise|else) %objects%", "<.+> ? %objects% : %objects%");
```

Comment:
Patterns should be on their own line (for readability IMO)

Reply:
Sorry, how would I acccept your changes?

Reply:
Oh, I didn’t actually propose any changes for you to accept. You can just edit the file and do what I suggested though, and mark the conversation as resolved after you commit.

Reply:
Okay, thanks